### PR TITLE
(dotnet) Remove comments pertaining to auto-generation in Schema and Connector

### LINKED ISSUE #4946: All the comments about files being "auto-generated" in the Connector and Schema namespaces should be removed
## Version
What package version of the SDK are you using: **main** (4.11.0, as of this writing)

## Describe the bug
The files in the Microsoft.Bot.**Connector** and Microsoft.Bot.**Schema** namespaces are no longer being auto-generated from the Swagger file. However, most of the files still contain the `// <auto-generated>` comments. These comments should be removed and all the resulting code analysis issues resolved.

## To Reproduce
1. Open one such file, such as Microsoft.Bot.Connector/Attachments.cs
1. See error: https://github.com/microsoft/botbuilder-dotnet/blob/804380f5f59f9a4d4ece0ba4de788ffb4cb3f09b/libraries/Microsoft.Bot.Connector/Attachments.cs#L1-L11

## Expected behavior
Since these are no longer auto-generated, this auto-generated auto-generated comment should be removed from these and other such files.

## Additional context
1. This will not be a trivial process, as code analysis is turned off for auto-generated files.
1. This issue appears to be present in the JavaScript and Python SDKs, too. I have not checked on the Java SDK.
1. This comment occurs in other namespaces, but I'm not certain which other ones, if any, are labeled such in error.

### PR DESCRIPTION
**Pull request by Zerryth**
Fixes #4946 
Partially fixes [bf-sdk issue#5887](https://github.com/microsoft/botframework-sdk/issues/5887)

## Description
Files in `Microsoft.Bot.Connector` and `Microsoft.Bot.Schema` are no longer auto-generated from Swagger, however most files still contain `// <auto-generated>` comments. This PR:
- Removes "auto-generated" comments in Connector and Schema
    - Replace comment with standard Copyright header
    ```
    // Copyright (c) Microsoft Corporation. All rights reserved.
    // Licensed under the MIT License.
    ```
- Removes misleading `Generated` folder in `Microsoft.Bot.Schema/Teams/`
    - Files in this subfolder moved to be under `Teams/` folder directly
___
Update: This PR also corrects 1800+ styling errors that were hidden previously when the files were considered "auto-generated", but now had unearthed themselves for a great big surprise Christmas present to me 😵  ✨ 🎉 😅 🎁

### CODE REVIEW

**Comment by Zerryth** [comment]
Uh wow lots of styling rules broken in these files, not necessarily having to do with the Copyright comments added. I'll clean those up so tests won't break...

**Comment by mrivera-ms** [comment]
@Zerryth the checks are still failing. Can you please look at them and fix whatever is needed? Thanks.

**Comment by Zerryth** [comment]
@mrivera-ms yeah, slogging through them. I'm not sure which it was--either removing the <auto-generated> tag or moving the files out of Generated file, but originally they were not checked against stylecop, and now they are, so ALL the files are throwing stylecop errors, which are 1000+ since these "auto-generated" files have been going unchecked for style for so long. 😵 

Slooooowly making my way through them, correcting the style.

These are some of the styling errors so far:
* Change encoding from 'utf-8' to UTF-8 with byte order mark
* Periods in documentation
* Remove blank line after curly brace
* System directives should be placed before other using directives
* Constructor summary documentation should begin with standard text
* Element documentation should have a summary.

Right now the errors that are the most work-intensive are ones that require properties to have a `<value>` tag with text detailing the value of all the formerly auto-generated code.

**Comment by Zerryth** [comment]
Ok, winnowed down a huge chunk of the errors so far:
**Before**:
image

Still have 600+ to go.
**Current**:
image

Here's a small snippet of some of the style errs. remaining:
image

Nothing show-stopping or anything.
I hate jumping between issues, however I'm going to put a pin on working on this one right now, to hop onto a P0 issue fix first, but will come back to this after.

**Comment by Zerryth** [comment]
Alright, @mrivera-ms ! Cleared the rest of the styling error detritus that remained from the formerly auto-generated code, and it should *actually* be ready for review now. 

<details>
<summary>Click to view some additional styling errors corrected.</summary>

* Local variables needing to start with a lower-case letter (a lot of auto-generated ones started with `_`)
* The behavior of '`string.EndsWith(string)`' could vary based on the current user's locale settings. Replace this call...with a call to '`string.EndsWith(string, System.StringComparison)`'
* The behavior of '`long.ToString()`' could vary based on the current user's locale settings. Replace this...with a call to '`long.ToString(IFormatProvider)`'
* The behavior of '`string.Format(string, object)`' could vary based on the current user's locale settings. Replace this...with a call to '`string.Format(IFormatProvider, string, params object[])`'
* Use `string.Empty` instead of `""`

</details>

File: libraries/Microsoft.Bot.Connector/Attachments.cs [hunk 1]
```diff
@@ -66,152 +62,168 @@ public Attachments(ConnectorClient client)
...
-        /// Thrown when the operation returned an invalid status code
+        /// Thrown when the operation returned an invalid status code.
         /// </exception>
         /// <exception cref="SerializationException">
-        /// Thrown when unable to deserialize the response
+        /// Thrown when unable to deserialize the response.
         /// </exception>
         /// <exception cref="ValidationException">
-        /// Thrown when a required parameter is null
+        /// Thrown when a required parameter is null.
```
Comment (mrivera-ms) [inline libraries/Microsoft.Bot.Connector/Attachments.cs:71]: >Thrown when a required parameter is null. [](start = 12, length = 41)

This is thrown only when attachment Id is null. If you update this comment, you can remove the pragma that disables SA1625

File: libraries/Microsoft.Bot.Connector/Attachments.cs [hunk 2]
```diff
@@ -66,152 +62,168 @@ public Attachments(ConnectorClient client)
...
             {
-                ServiceClientTracing.Exit(_invocationId, _result);
+                ServiceClientTracing.Exit(invocationId, result);
             }
-            return _result;
+
+            return result;
         }
 
+#pragma warning disable SA1625 // Element documentation should not be copied and pasted
```
Comment (mrivera-ms) [inline libraries/Microsoft.Bot.Connector/Attachments.cs:215]: >#pragma warning disable SA1625  [](start = 0, length = 31)

Please add a more detailed param descriptions and remove these pragmas.

File: libraries/Microsoft.Bot.Connector/BotSignIn.cs [hunk 3]
```diff
@@ -1,220 +1,223 @@
...
-        /// <exception cref="ValidationException">
-        /// Thrown when a required parameter is null
-        /// </exception>
-        /// <exception cref="System.ArgumentNullException">
-        /// Thrown when a required parameter is null
-        /// </exception>
-        /// <return>
-        /// A response object containing the response body and response headers.
-        /// </return>
+#pragma warning disable SA1625 // Element documentation should not be copied and pasted
```
Comment (mrivera-ms) [inline libraries/Microsoft.Bot.Connector/BotSignIn.cs:49]: >pragma warning disable SA1625  [](start = 1, length = 30)

Remove.

File: libraries/Microsoft.Bot.Connector/Attachments.cs (hunk 2 above)
Comment (Zerryth) [reply libraries/Microsoft.Bot.Connector/Attachments.cs]: Done! Docs updated -- removed `#pragma warning disable SA1625`

File: libraries/Microsoft.Bot.Connector/BotSignIn.cs (hunk 3 above)
Comment (Zerryth) [reply libraries/Microsoft.Bot.Connector/BotSignIn.cs]: Done! Docs updated -- removed `#pragma warning disable SA1625`

File: libraries/Microsoft.Bot.Schema/TokenExchangeResource.cs [hunk 4]
```diff
@@ -1,34 +1,30 @@
...
         }
 
         /// <summary>
-        /// Initializes a new instance of the TokenExchangeResource class.
+        /// Initializes a new instance of the <see cref="TokenExchangeResource"/> class.
         /// </summary>
+        /// <param name="id">ID.</param>
+        /// <param name="uri">URI.</param>
+        /// <param name="providerId">Identity provider ID.</param>
         public TokenExchangeResource(string id = default(string), string uri = default(string), string providerId = default(string))
```
Comment (mrivera-ms) [inline libraries/Microsoft.Bot.Schema/TokenExchangeResource.cs:28]: Please add better descriptions.

**Review by mrivera-ms — APPROVED** [review]
:shipit:

**Pull request merged**